src/main/features/core/windowTitleInfo.js

```diff
@@ -0,0 +1,16 @@
+const windowTitle = WindowManager.get(global.mainWindowID).getTitle();
+let songInfo = {};
+
+if (Settings.get('enableWindowTitleInfo')) {
+  PlaybackAPI.on('change:song', (_songInfo) => {
+    songInfo = _songInfo;
```

Comment:
Yep, I would prefer using `PlaybackAPI.getCurrentSong()` (or whatever it actually is)